src/Rules/Doctrine/ORM/PropertiesExtension.php

```diff
@@ -43,6 +44,10 @@ public function isAlwaysWritten(PropertyReflection $property, string $propertyNa
 			return false;
 		}
 
+		if ($this->isGeneratedIdentifier($metadata, $propertyName)) {
```

Comment:
This exact thing is already checked in this method. I believe if you remove these lines, all tests should still pass.